README.md

```diff
@@ -13,67 +13,66 @@
 | appconfiguration | [spring-cloud-azure-feature-management:1.3.0]                                                  | [feature-management-sample](appconfiguration/feature-management-sample)                                                           |
 | appconfiguration | [spring-cloud-azure-feature-management:1.3.0]                                                  | [feature-management-web-sample](appconfiguration/feature-management-web-sample)                                                   |
 | appconfiguration | [spring-cloud-starter-azure-appconfiguration-config:1.3.0]                                     | [azure-appconfiguration-conversion-sample-complete](appconfiguration/azure-appconfiguration-conversion-sample-complete)           |
 | cache            | N/A                                                                                            | [azure-spring-cloud-sample-cache](cache/spring-cloud-azure-starter/spring-cloud-azure-sample-cache)                                                          |
-| cloudfoundry     | [azure-spring-boot-starter-cosmos:3.9.0]<br />[azure-spring-boot-starter-servicebus-jms:3.9.0] | [azure-cloud-foundry-service-sample](cloudfoundry/azure-cloud-foundry-service-sample)                                             |
-| cosmos           | [azure-spring-boot-starter-cosmos:3.9.0]                                                       | [cosmos-multi-database-multi-account](cosmos/azure-spring-data-cosmos/cosmos-multi-database-multi-account)                |
-| cosmos           | [azure-spring-boot-starter-cosmos:3.9.0]                                                       | [cosmos-multi-database-single-account](cosmos/azure-spring-data-cosmos/cosmos-multi-database-single-account)              |
-| cosmos           | [azure-spring-boot-starter-cosmos:3.9.0]                                                       | [spring-cloud-azure-data-cosmos-sample](cosmos/spring-cloud-azure-starter-data-cosmos/spring-cloud-azure-data-cosmos-sample)                                                                          |
-| cosmos           | [azure-spring-boot-starter-cosmos:3.9.0]                                                       | [spring-cloud-azure-cosmos-sample](cosmos/spring-cloud-azure-starter-cosmos/spring-cloud-azure-cosmos-sample)                                                                          |
+| cloudfoundry     | N/A                                                                                            | [azure-cloud-foundry-service-sample](cloudfoundry/azure-cloud-foundry-service-sample)                                                          |
+| cosmos           | [azure-spring-data-cosmos:3.13.1]                                                              | [cosmos-multi-database-multi-account](cosmos/azure-spring-data-cosmos/cosmos-multi-database-multi-account)                |
```

Comment:
Why keep `azure-spring-data-cosmos` and delete `azure-security-keyvault-jca`?

Reply:
1. this is azure-spring-data-cosmos, no released in 4.x
2. run-with-command-line-server-side doesn't depend on azure-security-keyvault-jca